cmd/state-offline-installer/main.go

```diff
@@ -24,11 +25,22 @@ import (
 	"github.com/ActiveState/cli/pkg/cmdlets/errors"
 )
 
+type Params struct {
+	path string
+}
+
+func newParams() *Params {
+	return &Params{
+		path: "/tmp",
```

Comment:
I would prefer a more OS-agnostic value here, even if it's just a placeholder.

Reply:
Yeah in fact it shouldn't be here at all. It's not how defaults work.